src/components/Dropzone.js

```diff
@@ -10,19 +10,19 @@ class DropzoneWrapper extends Component {
     };
   }
 
-  handleDropFile(accepted, rejected) {
+  handleDropFile = (accepted, rejected) => {
     const { handleDropFile, handleRejectDropped } = this.props;
 
     this.handleDragEnd();
 
-    if (accepted.length) {
-      handleDropFile(accepted);
+    for (const file of accepted) {
```

Comment:
I'm a fan of using functional style and would go with map instead. For-of is a huge overhead for a simple array.

Reply:
`map` returns an array that we are not using here, so `forEach` would reveal the intention a bit more.

Personally I think the `for-of` makes for really readable code, but I'm fine using either